build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
   compileOnly "org.projectlombok:lombok:${lombokVersion}"
   annotationProcessor "org.projectlombok:lombok:${lombokVersion}"
 
+  testCompile 'com.opentable.components:otj-pg-embedded:0.13.1'
```

Comment:
We no longer depend on `otj-pg-embedded` for an embedded postgres instance (replaced by [`MarquezDb`](https://github.com/MarquezProject/marquez/blob/master/src/test/java/marquez/MarquezDb.java)) . So, this dependency can be removed.

Reply:
actually the TagDaoTest still need this component, Would you like me to remove this test ?

Reply:
I'd prefer that we update the test instead: https://github.com/MarquezProject/marquez/pull/664#pullrequestreview-327128524